### PR TITLE
Issue 366 post data length

### PR DESCRIPTION
**Pull request by jjeane**
### What is the context of this PR?

Post data payload limited to 64k
### How to review

Run eq-survey-runner
Check you can't submit more than 64k
- [ ] Do all commits have sensible commit messages?
- [ ] Is all new code unit tested?
- [ ] Are there integration tests if needed?
- [ ] Is there sufficient logging?
- [ ] Is there documentation or is the code self-describing?
### Who worked on the PR

@jjeane & @warrenbailey

### CODE REVIEW

File: app/schema/widgets/textarea_widget.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def render(self, state):
                 'label': state.schema_item.label,
                 'value': state.input or '',
                 'placeholder': '',
-                'maxChars': 2000,
+                'maxChars': state.schema_item.max_chars or 2000,
```
Comment (LJBabbage) [inline app/schema/widgets/textarea_widget.py]: We seem to still be using magic numbers, we need to get away from the 2000 and stick it as a default higher up

**Comment by warrenbailey** [comment]
There is already a section in the schema called max_length we can use. However for some reason it's not parser by  the schema parser. We should just correct schema parser and use that value instead:

Have a look at 01_0203.json:

```
"display": {
  "properties": {
    "max_length": "2000"
   }
},
```

File: app/main/views/questionnaire.py [hunk 2]
```diff
@@ -77,6 +77,9 @@ def do_get(questionnaire_manager, location):
 
 def do_post(collection_id, eq_id, location, questionnaire_manager):
     logger.debug("POST request question - current location %s", location)
+
+    logger.error("POST request length %s", request.content_length)
```
Comment (warrenbailey) [inline app/main/views/questionnaire.py]: maybe change to debug oops

**Comment by LJBabbage** [comment]
Think we need to redirected correctly on error.

**Pull request merged**